project.properties

```diff
@@ -2,6 +2,6 @@
 #Manifest merger has been enabled to pull in kit resources.
 #Tue Mar 10 09:55:06 GMT 2015
 manifestmerger.enabled=true
-target=android-21
+target=android-26
```

Comment:
I believe the play store is going to enforce higher sdk targets at some point. might as well target latest, 34